pkg/trace/api/normalizer_test.go

```diff
@@ -443,6 +445,8 @@ func TestNormalizeTag(t *testing.T) {
 			}(),
 			out: "a", // 'b' should have been truncated
 		},
+		{"a" + string(unicode.ReplacementChar), "a"},
+		{"a" + string(unicode.ReplacementChar) + string(unicode.ReplacementChar), "a"},
```

Comment:
A better test case would be to have valid letters after a replacement char and ensure the output contains them no?

Reply:
Indeed, I added it.